data/json/recipes/recipe_others.json

```diff
@@ -7904,5 +7904,36 @@
     [ [ "wheel", 1 ] ],
     [ [ "motor", 1 ] ]
   ]
+},{
+  "type" : "recipe",
+  "result": "neoprene",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_OTHER",
+  "skill_used": "cooking",
+  "skills_required": [ "cooking", 4 ],
+  "difficulty": 4,
```

Comment:
This recipe should probably take more skill (somewhere near 6 instead of 4) and time. I don't think that synthesizing neoprene via a multi-step chemical reaction is simple or fast.